crates/typst-syntax/src/set.rs

```diff
@@ -39,54 +39,61 @@ const fn bit(kind: SyntaxKind) -> u128 {
     1 << (kind as usize)
 }
 
+/// Generate an inline const syntax set of the given kinds.
+///
+/// Example: `if syntax_set!(Plus, Minus).contains(SyntaxKind::Plus) { ... }`
+/// Becomes:
+/// ```rust
+/// if {const SET:SyntaxSet = SyntaxSet::new()
+///     .add(SyntaxKind::Plus)
+///     .add(SyntaxKind::Minus)
+/// }.contains(SyntaxKind::Plus) { ... }
```

Comment:
I believe `SET` should be returned here within the block expression.

Reply:
Yep :upside_down_face: